File: VB/HowToAccessSchedulerControl.Module.Web/WebSchedulerController.vb
Imports System
Imports System.ComponentModel
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Text

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.Persistent.Base

Imports DevExpress.ExpressApp.Scheduler.Web
Imports DevExpress.Web.ASPxScheduler
Imports DevExpress.XtraScheduler

Imports DevExpress.Persistent.Base.General

Namespace HowToAccessSchedulerControl.Module.Web
    Public Class WebSchedulerController
        Inherits ObjectViewController(Of ListView, IEvent)

        Protected Overrides Sub OnViewControlsCreated()
            MyBase.OnViewControlsCreated()
            Dim listEditor As ASPxSchedulerListEditor = TryCast(View.Editor, ASPxSchedulerListEditor)
            If listEditor IsNot Nothing Then
                Dim scheduler As ASPxScheduler = listEditor.SchedulerControl
                scheduler.Views.DayView.VisibleTime = New TimeOfDayInterval(New TimeSpan(8, 0, 0), New TimeSpan(17, 0, 0))
            End If
        End Sub
    End Class
End Namespace

File: CS/HowToAccessSchedulerControl.Module.Web/WebSchedulerController.cs
using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Diagnostics;
using System.Text;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.Persistent.Base;

using DevExpress.ExpressApp.Scheduler.Web;
using DevExpress.Web.ASPxScheduler;
using DevExpress.XtraScheduler;

using DevExpress.Persistent.Base.General;

namespace HowToAccessSchedulerControl.Module.Web {
    public class WebSchedulerController : ObjectViewController<ListView, IEvent> {
        protected override void OnViewControlsCreated() {
            base.OnViewControlsCreated();
            ASPxSchedulerListEditor listEditor = View.Editor as ASPxSchedulerListEditor;
            if (listEditor != null) {
                ASPxScheduler scheduler = listEditor.SchedulerControl;
                scheduler.Views.DayView.VisibleTime =
                    new TimeOfDayInterval(new TimeSpan(6, 0, 0), new TimeSpan(11, 0, 0));
            }
        }
    }
}
